File: python/large_3number.py
num1=int(input("Tell me 1st Number: "))
num2=int(input("Tell me 2nd Number: "))
num3=int(input("Tell me 3rd Number: "))
if num1>num2 and num1>num3:
    print(num1,"is greater than",num2,"and",num3)
elif num2>num1 and num2>num3:
    print(num2,"is greater than",num1,"and",num3)
elif num3>num1 and num3>num2:
    print(num3,"is greater than",num1,"and",num2)
else:
    print("all are same")

File: c/large_3number.c
#include <stdio.h>
int main()
{
    int num1,num2,num3;
    printf("Tell me your 1st digit\n");
    scanf("%d",&num1);
    printf("Tell me your 2nd digit\n");
    scanf("%d",&num2);
    printf("Tell me your 3rd digit\n");
    scanf("%d",&num3);

    if (num1>num2 && num1>num3)
    {
        printf("%d is greater than %d and %d",num1,num2,num3);
    }
    else if (num2>num1 && num2>num3)
    {
        printf("%d is greater than %d and %d",num2,num1,num3);
    }
    else if (num3>num1 && num3>num2)
    {
        printf("%d is greater than %d and %d",num3,num1,num2);
    }
    else
    {
        printf("All are same");
    }
}